t/id-keyword-draft6.t

```diff
@@ -0,0 +1,45 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+use JSON::Validator;
+
+my ($base_url, $jv, $t, @e);
+
+use Mojolicious::Lite;
+get '/person'           => 'person';
+get '/invalid-relative' => 'invalid-relative';
+
+$t = Test::Mojo->new;
+$jv = JSON::Validator->new(ua => $t->ua);
```

Comment:
I don't think the code (any of the files in this PR) is run through perltidy. Please do so, using the perltidyrc found in this project.